Source/Core/Core/IOS/ES/Formats.h

```diff
@@ -219,8 +219,8 @@ class UIDSys final
 public:
   explicit UIDSys(Common::FromWhichRoot root);
 
-  u32 GetUIDFromTitle(u64 title_id);
-  void AddTitle(u64 title_id);
+  u32 GetUIDFromTitle(u64 title_id) const;
+  u32 AddTitle(u64 title_id);
```

Comment:
That's indeed how it's implemented. On second thought, keeping the name "AddTitle" is indeed a bit confusing. I'll rename it to make it clear that the UID is returned.